Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:20.18.0
+
+WORKDIR /app
+
+RUN npm install -g pnpm
+
+# Copy the rest of your app's source code
+COPY . .
+
+# Install dependencies
+RUN pnpm install
```

Comment:
Just my 50 cents (I'm not a maintainer of this repo) but you can [optimize your Dockerfile](https://docs.docker.com/build/cache/optimize/) further by taking advantage of cache layers:

```suggestion
RUN npm install -g pnpm

# Install dependencies (this step is cached as long as the dependencies don't change)
COPY package.json package-lock.json .
RUN pnpm install

# Copy the rest of your app's source code
COPY . .
```

Reply:
@sleeyax We can use corepack to enable the use of pnpm and it will save space instead of installing it globally.

```yaml
RUN corepack enable pnpm
```

Reply:
> Just my 50 cents (I'm not a maintainer of this repo) but you can [optimize your Dockerfile](https://docs.docker.com/build/cache/optimize/) further by taking advantage of cache layers:

Unfortunately, I am also not a maintainer but would love to become one.